domains/service/src/lib.rs

```diff
@@ -90,9 +90,11 @@ where
     if client.requires_full_sync() {
         match config.network.sync_mode {
             SyncMode::LightState { .. } => {
-                return Err("Fast sync doesn't work for archive nodes".into());
+                return Err("Fast sync doesn't work for archive nodes: use full sync".into());
```

Comment:
Messages in this file do not need to be changes as they are not really actually used. The message you're getting on devnet is printed by Substrate's upstream code (this is a copy-paste of it with some changes for domains, so message looks the same, but it is actually not used).